Formula/g/gstreamer.rb

```diff
@@ -249,6 +249,13 @@ def post_install
     libnice_gst_plugin = Formula["libnice-gstreamer"].opt_libexec/"gstreamer-1.0"/shared_library("libgstnice")
     gst_plugin_dir = lib/"gstreamer-1.0"
     ln_sf libnice_gst_plugin.relative_path_from(gst_plugin_dir), gst_plugin_dir
```

Comment:
I wonder if we should just have essentially a postinstall hook that generates reverse symlinks so that a user can install systemwide plugins and then run `brew postinstall gstreamer`.